src/hooks/usePriceFetcher.ts

```diff
@@ -66,8 +68,8 @@ const assets: PriceFetcherAsset[] = [
   {
     assetName: 'AMPE',
     blockchain: 'Amplitude',
-    assetId: undefined,
-    provider: 'dia',
+    assetId: 'eth',
```

Comment:
No need to call it `eth`, we can also call it 'ampe' here I guess.

Reply:
```suggestion
    assetId: 'eth',
    assetId: 'ampe',
```